packages/template-retail-react-app/package.json

```diff
@@ -12,12 +12,13 @@
     "analyze-build": "cross-env MOBIFY_ANALYZE=true npm run build",
     "build": "npm run build-translations && pwa-kit-dev build",
     "build-translations": "npm run extract-default-translations && npm run compile-translations",
-    "compile-translations": "formatjs compile-folder --ast translations translations/compiled",
-    "compile-translations:pseudo": "formatjs compile --ast translations/en-US.json --out-file translations/compiled/en-XA.json --pseudo-locale en-XA",
+    "compile-translations": "node ./scripts/compile-translations.js translations",
+    "compile-translations:pseudo": "node ./scripts/compile-pseudo-translation.js translations/en-US.json",
     "extract-default-translations": "node ./scripts/extract-default-messages.js en-US",
     "format": "pwa-kit-dev format \"**/*.{js,jsx}\"",
     "lint": "pwa-kit-dev lint \"**/*.{js,jsx}\"",
     "lint:fix": "npm run lint -- --fix",
+    "postinstall": "npm run compile-translations && npm run compile-translations:pseudo",
```

Comment:
Sorry, i'm not sure if i understand your comment. Doesn't `npm start` or `npm run build` compiles the translations? why do you need the translations post install?

Reply:
@kevinxh does running `npm start` really compile the translations? I didn't see that happening on my end. I was seeing the server running into infinite loop and not able to load the site because of the missing translations.

When generating a project, the bootstrap assets do not include the compiled translation files. This is intentional to have a minimal bootstrap.

Right at the end of project generation, we run this `npm install` [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/8cd3b3ddd244693d9cf63c35d0a1fa09744ed897/packages/pwa-kit-create-app/scripts/create-mobify-app.js#L620). So afterwards, the `postinstall` script would run too, making sure that the compiled translation files are in the static folder.

Reply:
> I was seeing the server running into infinite loop and not able to load the site because of the missing translations.

Some thoughts... (I feel like i might miss some knowledge regarding the translation stuff, so pls correct me)
1. it sounds like we need to compile translations while developing locally? so it makes sense to me that the `npm start` also compiles translations? what if someone accidentally deleted the `/static` folder after project generation?
2. we need error handling so if server fails to fetch, it doesn't be in an infinite loop?

Also, this isn't a blocking issue, so feel free to ignore me.